baybe/parameters/numerical.py

```diff
@@ -150,5 +150,38 @@ def summary(self) -> dict:
         return param_dict
 
 
+@define(frozen=True, slots=False)
+class _FixedNumericalContinuousParameter(ContinuousParameter):
+    """Parameter class for fixed numerical parameters."""
+
+    is_numeric: ClassVar[bool] = True
+    # See base class.
+
+    value: float = field(converter=float)
+    """The fixed value of the parameter."""
+
+    @property
+    def bounds(self) -> Interval:
+        """The value of the parameter as a degenerate interval."""
+        return Interval(self.value, self.value)
+
+    @override
+    def is_in_range(self, item: float) -> bool:
+        return item == self.value
```

Comment:
Do we want an exact match or here or some sort of a fuzzy match/is almost equal to? Or are such potential inaccuracies handled at another place?

Reply:
Hm, haven't really thought about it. I think, right now we actually don't need to bother because there is yet no use for such a comparison (in particular, because the class in used only internally). How about we think about it once we have the first actual use case for it and then decide based on that what would be the appropriate thing to do? We can easily add an optional `tolerance`  instance attribute with default of `0.0` later